src/Pages/OfficerDB/App.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import './App.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { Table } from 'reactstrap'
+
+export default class App extends React.Component {
```

Comment:
Please also write test cases for this component. You can use [this example](https://github.com/SCE-Development/Core-v4/blob/7eb49330115efe0619c97377f7b69382cca2d41d/test/frontend/Slideshow.test.js#L13) for reference.